.github/workflows/lighthouse.yml

```diff
@@ -12,32 +12,43 @@ on:
 
 # A workflow run is made up of one or more jobs that can run sequentially or in parallel
 jobs:
-  # This workflow contains a single job called "build"
-  build:
+  # This workflow contains a single job
+  lighthouse:
     # The type of runner that the job will run on
     runs-on: ubuntu-latest
 
     # Steps represent a sequence of tasks that will be executed as part of the job
     steps:
     # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
     - uses: actions/checkout@v2
+    - name: Setup Node
+      uses: actions/setup-node@v1
 
-    # Runs a single command using the runners shell
-    - name: Run a one-line script
-      run: echo Hello, world!
+    - name: NPM install and Build
+      run: |
+        npm i
+        npm run build
+
+    - name: Waiting for 200 from the Netlify Preview
+      uses: jakepartusch/wait-for-netlify-action@v1
+      id: waitFor200
+      with:
+        site_name: "happy-franklin-69b6d4"
 
-    # Runs a set of commands using the runners shell
-    - name: Run a multi-line script
+    # lighthouse localhost test
```

Comment:
@sniperadmin This seems to pull and install lighthouse on the local CI environment , but I don't see how it runs tests against a locally deployed ambianic UI instance? Can you tag the lines that start the local UI instance and then point lighthouse to it? I don't see "npm run serve" nor reference to localhost:8080. What am I missing?

Reply:
this will need some more commits i guess